config/routes.rb

```diff
@@ -29,6 +29,7 @@
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
   get '/auth/github/callback', to: 'sessions#update'
+  get '/logout', to: 'sessions#destroy'
```

Comment:
I added this in because I frequently go to localhost and there is a bad user logged in. This will delete the sessions. I dont believe its restful as there is already a path to 'sessions#destroy'. Wouldnt mind talking about this

Reply:
I'm a little confused by this so yeah, let's talk